test/SILGen/destroy_enum_in_all_cases.swift

```diff
@@ -0,0 +1,45 @@
+// RUN: %target-swift-emit-silgen -module-name switch %s > %t.sil
+// RUN: %target-sil-opt %t.sil -sil-combine -enable-sil-verify-all | %FileCheck %s
+
```

Comment:
This test crashes on master. It's a bit unconventional so, I want to get this re-approved before I merge anything.